doc/1-intro/glossary.md

```diff
@@ -16,10 +16,13 @@
 
 **bitbang**: to achieve PCM sound by sending a rapid stream of volume commands to a non-PCM channel.
 
+**bit depth**: the number of bits used for each sample in a PCM stream. 16-bit is considered high quality; lower bit depths introduce more noise and distortion, especially for quieter samples.
+- the "1-bit DPCM" sample type has 6-bit amplitude resolution, but each sample is represented by only 1 bit indicating whether its amplitude differs from the previous sample by one step up or down.
```

Comment:
7-bit resolution (???) - please confirm
Change "amplitude" to "value". We can explain PCM later.